### PR TITLE
Validate files now works

### PR DESCRIPTION
**Pull request by jochman**
thanks to my deep night sleep

## Ready

### CODE REVIEW

**Comment by liorblob** [comment]
72 files changed?? Why??

File: demisto_sdk/common/hook_validations/base_validator.py [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import re
+from abc import abstractmethod
+
+from demisto_sdk.common.constants import Errors
+from demisto_sdk.common.hook_validations.structure import StructureValidator
+from demisto_sdk.common.tools import print_error, get_release_notes_file_path, \
+    get_latest_release_notes_text, run_command
+
+
+class BaseValidator(object):
```
Comment (liorblob) [inline demisto_sdk/common/hook_validations/base_validator.py]: remove the `(object)`

File: demisto_sdk/common/hook_validations/base_validator.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+    DEFAULT_VERSION = -1
+
+    def __init__(self, structure_validator):
```
Comment (liorblob) [inline demisto_sdk/common/hook_validations/base_validator.py]: this is python 3 now, types should be on the arguments
add docstrings

File: demisto_sdk/common/hook_validations/id.py [hunk 3]
```diff
@@ -4,7 +4,7 @@
 from distutils.version import LooseVersion
 from collections import OrderedDict
 
-from demisto_sdk.common.configuration import Configuration
+from demisto_sdk.validation.configuration import Configuration
```
Comment (liorblob) [inline demisto_sdk/common/hook_validations/id.py]: why is it in validation?

File: demisto_sdk/common/hook_validations/structure.py [hunk 4]
```diff
@@ -1,310 +1,234 @@
...
+    MISC_REGEX, Errors, ACCEPTED_FILE_EXTENSIONS
+from demisto_sdk.common.tools import get_remote_file, get_matching_regex, print_error
+from demisto_sdk.validation.configuration import Configuration
 
-from demisto_sdk.common.configuration import Configuration
-# TODO: do not import *
-from demisto_sdk.common.constants import *
-from demisto_sdk.common.tools import print_error, print_warning, run_command, get_yaml, get_json, checked_type, \
-    get_release_notes_file_path, get_latest_release_notes_text
+try:
```
Comment (liorblob) [inline demisto_sdk/common/hook_validations/structure.py]: DELETE THIS

File: test-requirements.txt [hunk 5]
```diff
@@ -1,3 +1,5 @@
 pytest
+pytest-mock
 PyPDF2
-bs4
\ No newline at end of file
+bs4
+mock
```
Comment (liorblob) [inline test-requirements.txt]: remove

File: demisto_sdk/common/hook_validations/base_validator.py (hunk 2 above)
Comment (jochman) [reply demisto_sdk/common/hook_validations/base_validator.py]: it's valid also fro py3

File: test-requirements.txt (hunk 5 above)
Comment (jochman) [reply test-requirements.txt]: y? our tests must run on this

Comment (liorblob) [reply test-requirements.txt]: it needs to be in `requirements.txt`

**Pull request merged**